manuscript/040_Three_Essential_Tools.md

```diff
@@ -154,28 +154,28 @@ Multiplication_ShouldResultInAMultiplicationOfTwoPassedNumbers()
+3.  The **Assertion** takes the form of a call to the static `Assert.Equal()` method -- the xUnit.NET framework is bundled with a wide range of assertion methods. Of course, no one stops you from writing your own if the provided assertion methods do not offer what you are looking for.
 
-Phew, I hope I made the transition quite painless for you. Now the last thing to add -- as there is not `Main()` method anymore in the last example, you surely must wonder how we run those tests, right? Ok, the last big secret unveiled -- we use an external application for this (we will refer to it using a term **Test Runner**) -- we tell it which assemblies to run and it loads them, runs them, reports results etc. It can take various forms, e.g. it can be a console application, a GUI application or a plugin to our IDEs. Here is an example of a stand-alone runner for xUnit.NET framework:
+Phew, I hope I made the transition quite painless for you. Now the last thing to add -- as there is no `Main()` method anymore in the last example, you surely must wonder how we run those tests, right? Ok, the last big secret unveiled -- we use an external application for this (we will refer to it using the term **Test Runner**) -- we tell it which assemblies to run and it loads them, runs them, reports results etc. It can take various forms, e.g. it can be a console application, a GUI application or a plugin for an IDE. Here is an example of a stand-alone runner for the xUnit.NET framework:
 
-![XUnit.NET window](images/XUnit_NET_Window.png)
+![xUnit.NET window](images/XUnit_NET_Window.png)
 
 Mocking framework
 -----------------
 
-Mocking libraries automate runtime creation of objects (called “mocks") that adhere to specified interface. Aside from the creation itself, the libraries provide an API to configure our mocks on how they behave when certain methods are called on them and to let us inspect which calls they received.
+A mocking framework lets us create objects at runtime (called “mocks") that adhere to a certain interface. That interface is specified when the mock is created. Aside from the creation, the framework provides an API to configure the mocks on how they behave when certain methods are called on them and to let us inspect which calls they received.
 
-Mocking libraries are not as old as xUnit frameworks and were not present in TDD at very beginning. As you might be wondering why on earth do we need something like this, I will let you in on a secret: they were born with a goal of aiding a specific approach to object-oriented design in mind. This kind of design is what TDD can support quite well as you will soon experience.
+Mocking frameworks are not as old as test frameworks and they were not present in TDD at the very beginning. Why on earth do we need something like this? Well, they where invented to support a specific approach to object-oriented design that is a very good fit to TDD as you soon will experience.
```

Comment:
I have some doubts about the "that is a very good fit to TDD" part. The intent was more that TDD can be adapted to seamlessly support this kind of design approach. As I read the sentence now, it seems to suggest that the design approach can be adapted to TDD.